src/server/http/xsrf.test.js

```diff
@@ -186,7 +181,7 @@ describe('xsrf request filter', function () {
         });
 
         expect(resp.statusCode).toBe(400);
-        expect(resp.result.message).toBe('Request must contain a kbn-xsrf header.');
+        expect(resp.result).toMatchSnapshot(`${method} reject response`);
```

Comment:
__note:__ I switched to snapshot here since `resp.result` is "raw" string now: request is proxied (and "abandoned") from core to the legacy Kibana and hence response isn't deserialized by the Hapi we use in core.